srcpkgs/android-tools/template

```diff
@@ -5,6 +5,7 @@ pkgname=android-tools
 #         curl -L http://git.io/vvC0Z | sh -s 5.0.2_r1 5.1.0_r1
 version=9.0.0r35
 revision=1
+archs="x86_64* i686*"
```

Comment:
Sure about this one? E.g. Alpine lists `x86 x86_64 aarch64 armv7 armhf`

Reply:
Edit: https://github.com/alpinelinux/aports/commit/67283cdbf6ef0ce2912bd28297714d6bb3976a38
Leave it as you done for now, my statement for nocross also applies for native arm, so we would need to set `-Wno-attributes` can test it somewhen later.